modular_skyrat/modules/sec_haul/code/guns/bullets.dm

```diff
@@ -2,37 +2,39 @@
 	icon = 'modular_skyrat/modules/sec_haul/icons/guns/projectiles.dmi'
 	icon_state = "bullet"
 
-////////////6mm///////////
-//LETHAL | IHDF | RUBBER//
+////////////6.3mm///////////
+//FLECHETTE | FRAGMENTING | DISSUASIVE//
 //////////////////////////
 /obj/item/ammo_casing/b6mm
-	name = "6mm bullet casing"
-	desc = "A 6mm bullet casing."
+	name = "6.3mm flechette casing"
+	desc = "A spent flechette."
 	icon = 'modular_skyrat/modules/sec_haul/icons/guns/ammo_cartridges.dmi'
 	icon_state = "sl-casing"
 	caliber = CALIBER_6MM
 	projectile_type = /obj/projectile/bullet/advanced/b6mm
 
 /obj/projectile/bullet/advanced/b6mm
```

Comment:
I suggest adding more functionality to different round types instead of number changes

Reply:
Those number and variable changes are their functionality. The right tool for the right job.
I don't need to make an entire new file with a new feature just for one gimmick caliber.